test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/ButtonSample.cs

```diff
@@ -28,186 +28,259 @@ public class ButtonSample : IExample
+            parent2.Add(iconButton);
+            iconButton.ClickEvent += (ojb, e) => {
+                var btn = iconButton.ButtonIcon.GetParent() as Button;
+                string name = btn.Name;
+            };
 
+            parent3 = new View()
+            {
+                Size = new Size(600, 400),
+                Layout = new LinearLayout()
+                {
+                    LinearAlignment = LinearLayout.Alignment.Top,
+                    LinearOrientation = LinearLayout.Orientation.Vertical,
+                    CellPadding = new Size(50, 50),
+                }
+            };
+
+            //Show a button with icon and text.
             iconTextButton = new Button();
-            iconTextButton.Style.BackgroundImage = CommonResource.GetTVResourcePath() + "component/c_buttonbasic/c_basic_button_white_bg_normal_9patch.png";
-            iconTextButton.Style.BackgroundImageBorder = new Rectangle(4, 4, 5, 5);
+            var iconTextStyle = iconTextButton.Style;
+            iconTextStyle.BackgroundColor = new Selector<Color>();
+            iconTextStyle.Text.Text = "IconTextButton";
+            iconTextButton.ApplyStyle(iconTextStyle);
+            iconTextButton.BackgroundImage = CommonResource.GetTVResourcePath() + "component/c_buttonbasic/c_basic_button_white_bg_normal_9patch.png";
+            iconTextButton.BackgroundImageBorder = new Rectangle(4, 4, 5, 5);
             iconTextButton.IconRelativeOrientation = Button.IconOrientation.Left;
-            iconTextButton.Style.Icon.ResourceUrl = CommonResource.GetTVResourcePath() + "component/c_radiobutton/c_radiobutton_white_check.png";
-            iconTextButton.Style.IconPadding.Top = 20;
-            iconTextButton.Style.IconPadding.Bottom = 20;
-            iconTextButton.Style.IconPadding.Start = 20;
-            iconTextButton.Style.IconPadding.End = 20;
-            iconTextButton.Style.Text.Text = "IconTextButton";
-            iconTextButton.Style.TextPadding.Top = 20;
-            iconTextButton.Style.TextPadding.Bottom = 20;
-            iconTextButton.Style.TextPadding.Start = 20;
-            iconTextButton.Style.TextPadding.End = 50;
-            iconTextButton.Size2D = new Size2D(500, 300);
-            iconTextButton.Position2D = new Position2D(900, 100);
-            root.Add(iconTextButton);
+            iconTextButton.ButtonIcon.ResourceUrl = CommonResource.GetTVResourcePath() + "component/c_radiobutton/c_radiobutton_white_check.png";
+            iconTextButton.IconPadding = new Extents(20, 20, 20, 20);
+            iconTextButton.TextPadding = new Extents(20, 50, 20, 20);
+            iconTextButton.Size = new Size(500, 300);
+            parent3.Add(iconTextButton);
 
             ///////////////////////////////////////////////Create by Property//////////////////////////////////////////////////////////
+            //Create utility basic style of button.
```

Comment:
I think this section is for "Create (a Button) by Property".
So it;d better not to use Style when create it.
Can you remove the code generating style, and use button's properties instead?
(e.g. Button.ButtonText, Button.ButtonIcon etc.)

Reply:
Style is for selector.
Keep unchanged